src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -257,8 +257,8 @@ public void storeContext(Context context, HttpServletRequest request) {
 		// Verify this is an LTI launch request and some of the required parameters (if not stub testing)
 		if( !isStubTesting ){
 			if ( ! SectionUtilityToolFilter.BASIC_LTI_LAUNCH_REQUEST.equals(request.getParameter(SectionUtilityToolFilter.LTI_MESSAGE_TYPE)) ||
-				  ! LTI_1P0_CONST.equals(request.getParameter(LTI_VERSION)) ||
-				  ltiKey == null) {
+					! LTI_1P0_CONST.equals(request.getParameter(LTI_VERSION)) ||
```

Comment:
I'm not 100% sure. When I was done with development I did the auto indent command in Eclipse and this is how it saved.